guide/english/book-recommendations/python/index.md

```diff
@@ -71,7 +70,6 @@ You'll learn many of the first princliples for data science tools and how algori
 
 - [Functional Programming in Python by David Mertz [OREILLY] [FREE]](https://www.oreilly.com/programming/free/files/functional-programming-python.pdf)
 - [Python in Education by Nicholas H. Tollervey[OREILLY] [FREE]](https://www.oreilly.com/programming/free/files/python-in-education.pdf)
-- [How to think like a Computer Scientist by Allen Downey[V 2.0.17] [FREE]](http://greenteapress.com/thinkpython/thinkpython.pdf)
```

Comment:
What's wrong with this one?

Reply:
This is a duplicate. The very first entry in this file(ISBN-13: 978-1491939369) is the same link.